### PR TITLE
[FIX][#1580]database_cleanup: Skip studio_customization module skippe…

### LINKED ISSUE #1580: database cleanup delete views and fields created by odoo studion!
I installed the module in odoo v10 then I cleaned the database after that I found out that the fields and views created with odoo studio not existing anymore!

### PR DESCRIPTION
**Pull request by bizzappdev**
…d for uninstalling

FIX #1580

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Review by ivantodorovich — APPROVED** [review]

File: database_cleanup/models/purge_modules.py [hunk 1]
```diff
@@ -67,6 +67,8 @@ def find(self):
         for module in IrModule.search([('to_buy', '=', False)]):
             if get_module_path(module.name, display_warning=False):
                 continue
+            if module.name == 'studio_customization':
```
Comment (sswapnesh) [inline database_cleanup/models/purge_modules.py]: Can't we add this directly on domain ?

(or atleast combine it with` if get_module_path(module.name, display_warning=False)` above)

Comment (bizzappdev) [reply database_cleanup/models/purge_modules.py]: adding into domain the negative condition is also an option.

Comment (bizzappdev) [reply database_cleanup/models/purge_modules.py]: Changes done

**Review by sswapnesh — APPROVED** [review]

**Comment by bizzappdev** [comment]
rebase done

**Comment by pedrobaeza** [comment]
And there's a linter error:
```
database_cleanup/models/purge_modules.py:68:80: E501 line too long (80 > 79 characters)
```

**Comment by bizzappdev** [comment]
@pedrobaeza  Pylint error solved.

**Pull request closed without merging**